backend/src/wire/user.rs

```diff
@@ -47,6 +47,31 @@ pub(crate) struct UserContent {
     pub status: MembershipStatus,
     #[validate(email)]
     pub email: String,
+    pub phone: String,
+    #[validate(range(
+        min = 0,
+        max = 9_999_999_999_i64,
+        message = "Student Number must only contain numbers"
+    ))]
+    pub student_number: Option<i64>,
+    #[validate(range(
+        min = 0,
+        max = 9_999_999_999_i64,
+        message = "NKBV Number must only contain numbers"
+    ))]
+    pub nkbv_number: Option<i64>,
+    #[validate(range(
+        min = 0,
+        max = 9_999_999_999_i64,
+        message = "Sportcard Number must only contain numbers"
+    ))]
+    pub sportcard_number: Option<i64>,
```

Comment:
I would propose to use a ˋu32ˋ, instead of an ˋi64ˋ.
This makes the validation somewhat superfluous. The Pg Datatype and SQL file would need to be adopted accordingly

Reply:
Okay, then might as well use i32 right? I've looked at the numbers and maximum is 9 numbers. Since the sql scheme normally only seems to have regular integer and not unsigned.

Reply:
Oh yeah, I forgot that Postgres only supports integers and not unsigned integers. A bit annoying. Do we ever have the case of leading zeros?

Reply:
Nope, should never be a problem!